gr_modtool template choose wrong MANIEST file to install

### What happened?

I was following [Creating Python OOT with gr-modtool tutorial](https://wiki.gnuradio.org/index.php?title=Creating_Python_OOT_with_gr-modtool)
This error occurs during `sudo make install` command: 
```
CMake Error at cmake_install.cmake:50 (file):
  file INSTALL cannot find
"/mydirectory/gr-OOT_module/MANIFEST.md": No
  such file or directory.
```

Then I  found:
According to [gnuradio/gr-utils/modtool/templates/gr-newmod](https://github.com/gnuradio/gnuradio/tree/3aae3dadb0e0d638f4e5b24d767b4d612cfa63ce/gr-utils/modtool/templates/gr-newmod), gr_modtool generates MANIFEST.yml for a new module.
However, when installing the module, [gnuradio/gr-utils/modtool/templates/gr-newmod/CMakeList.txt](https://github.com/gnuradio/gnuradio/blob/3aae3dadb0e0d638f4e5b24d767b4d612cfa63ce/gr-utils/modtool/templates/gr-newmod/CMakeLists.txt) wants MANIFEST.md instead. `

I created MANIFEST.md manually to fix the error. I expect the MANIFEST file to be managed automatically.







### System Information

OS: Ubuntu 22
GR Installation Method:  Source


### GNU Radio Version

3.10 (maint-3.10)

### Specific Version

v3.10.9.2-60-gb0766390

### Steps to Reproduce the Problem

create a new module with modtool:
```
gr_modtool newmod
ModuleName
cd gr-ModuleName
```
compile and install the module
```
mkdir build
cd build
cmake ..
make
sudo make install
```

### Relevant log output

_No response_

--- a/gr-utils/modtool/templates/gr-newmod/CMakeLists.txt
+++ b/gr-utils/modtool/templates/gr-newmod/CMakeLists.txt
@@ -152,6 +152,6 @@ configure_package_config_file(
     ${CMAKE_CURRENT_BINARY_DIR}/cmake/Modules/${target}Config.cmake
     INSTALL_DESTINATION ${GR_CMAKE_DIR})
 
-install(FILES MANIFEST.md
-    RENAME MANIFEST-${VERSION_MAJOR}.${VERSION_API}.${VERSION_ABI}${VERSION_PATCH}.md
+install(FILES MANIFEST.yml
+    RENAME MANIFEST-${VERSION_MAJOR}.${VERSION_API}.${VERSION_ABI}${VERSION_PATCH}.yml
     DESTINATION share/gnuradio/manifests/howto)

